src/lib/parseArgs/paParse.cpp

```diff
@@ -570,6 +570,17 @@ int paParse
     RETURN_ERROR("paLimitCheck");
   }
 
+  if ((s != -2) && (paConfigActions(false) == -1))
+  {
+    RETURN_ERROR("paConfigActions");
+  }
+
+  if ((s != -2) && (paConfigActions(false) == -1))
```

Comment:
This seems like a duplication of code.
Cut 'n paste error?   (`paConfigActions` is called twice)

Reply:
Fixed in e00dc42